api/scripts/report-service-errors.ts

```diff
@@ -16,10 +16,24 @@ async function main() {
     endTime: endDate.toDate().toISOString(),
     startTime: startDate.toDate().toISOString(),
   };
-  const serviceErrors = await errors(
+  var serviceErrors = await errors(
```

Comment:
Typically we use `let` instead of `var`.  It may not matter in this context, but it can lead to unintended variable scoping issues.